admin/src/js/services/session.js

```diff
@@ -114,6 +114,8 @@ const _ = require('lodash/core');
 
         init: checkCurrentSession,
 
+        checkCurrentSession: checkCurrentSession,
```

Comment:
I've defined another export here named `checkCurrentSession` so it conveys better the intent when calling `Session.checkCurrentSession()`
I couldn't find any `Session.init()` call in the admin app so I wasn't sure if I could simply rename that export

Reply:
Yes, same, I didn't find any usage of `Session.init()`, I'd remove it and just have `Session.checkCurrentSession()`